fenix/app/src/main/res/values-night/colors.xml

```diff
@@ -12,6 +12,7 @@
     <color name="fx_mobile_layer_color_2">@color/photonDarkGrey30</color>
     <!-- Search -->
     <color name="fx_mobile_layer_color_3">@color/photonDarkGrey80</color>
+    <color name="fx_mobile_layer_color_search">@color/photonDarkGrey80</color>
```

Comment:
I am not sure where to put this comment, but something with background colours in private mode is wrong, when I focus the toolbar the address bar and toolbar backgrounds blend in

Reply:
It's because we are missing the layerSearch for the private theme in styles.xml https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/styles.xml#245

Reply:
Thanks Gabriel